src/components/molecules/BenefitFilter.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+import { FilteredBenefitsCounter } from "../atoms/FilteredBenefitsCounter";
+
+/**
+ * This component displays three navigation buttons which filters out the result of benefits
+ */
+export function BenefitFilter(props) {
+  const handleClick = () => props.filter();
```

Comment:
How are we supposed to know which button was clicked exactly with nothing being passed in ? 
For each of these buttons for example I should have something like 

eligible button onClick callback
`() => props.onFilter("eligible")`

help button onClick callback
`() => props.onFilter("help")`

other button onClick callback

`() => props.onFilter("other")`